### PR TITLE
[Testing:Developer] Added Pronouns to Sample Data

### LINKED ISSUE #9157: Add Pronouns To Sample Data
**What problem are you trying to solve with Submitty**
Currently in the Manage Students Tab (Instructor View) for developers there is no testing data for pronouns.
image

**Describe the way you'd like to solve this problem**
Edit the yaml file to add pronouns to students.
In .setup/data/users there are a handful of .yml files that set data for a small number of users
Add the pronoun field to those files and update the script that parses those files when we do a fresh “vagrant up” or a “recreate sample courses”.
Any users that aren’t explicitly included in that small set of files are generated randomly.
There’s also a script to generate that random data.  We’ll want to have a variety of data for that field, including empty/blank values.
Add them with certain probabilities, such as (example)
25% empty
30% He/Him
30% She/Her
10% They/Them
5% some other pronouns

### PR DESCRIPTION
**Pull request by hrose6967**
### Please check if the PR fulfills these requirements:
closes #9157
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?

No data in sample students for pronouns

### What is the new behavior?
Sample students now have randomly generated pronouns

### CODE REVIEW

File: .setup/data/users/instructor.yml [hunk 1]
```diff
@@ -3,5 +3,6 @@ user_group: 1
 user_id: instructor
 anon_id: lokMH456u2As096
 user_familyname: Instructor
+user_pronouns: She/Her
```
Comment (ziesski) [inline .setup/data/users/instructor.yml:6]: I forgot to add comment the discussion we had during meeting, order should not matter, but would be ideal to have consistent user_pronoun label to always be below user_numeric_id. However, if it too much of a burden please feel free to mark it as resolved.

**Comment by ziesski** [comment]
Above comment on able to see column but not specific pronoun is due to not re-creating all sample course data.
Running this command  will recreate all sample course data. 
sudo bash /usr/local/submitty/GIT_CHECKOUT/Submitty/.setup/bin/recreate_sample_courses.sh 
Also check https://submitty.org/developer/development_instructions/index for more information

**Comment by mahi-pas** [comment]
@ziesski I changed the order of pronouns in the yml files as requested. Are there any more changes?

**Review by ziesski — APPROVED** [review]
This looks good to go / may need to edit cypress test (6 of them failing with student info panel - this should be definitely addressed for future PRs as well)

**Comment by bmcutler** [comment]
@mahi-pas
please resolve conflicts (I've merged a PR that re-indented the .yml files to pass new 
linting rules)

**Review by NiaHeermance — APPROVED** [review]
Recreated sample data and the pronouns are there when I log into each user! I approve.

**Pull request merged**